src/pudl/transform/vcerare.py

```diff
@@ -368,205 +364,3 @@ def _get_year(df, year):
             parquet_writer.write_table(
                 pa.Table.from_pandas(df, schema=schema, preserve_index=False)
             )
-
-
-def _load_duckdb_table():
-    """Load VCE RARE output table to duckdb for running asset checks."""
-    parquet_path = str(_get_parquet_path())
-    return duckdb.read_parquet(parquet_path)
-
-
-@asset_check(
-    asset=out_vcerare__hourly_available_capacity_factor,
-    blocking=True,
-    description="Check that row count matches expected.",
-)
-def check_rows(context: AssetCheckExecutionContext) -> AssetCheckResult:
-    """Check rows."""
-    logger.info("Check VCE RARE hourly table is the expected length")
-
-    # Define row counts for report years
-    row_counts_by_year = {
-        2019: 27287400,
-        2020: 27287400,
-        2021: 27287400,
-        2022: 27287400,
-        2023: 27287400,
-    }
-
-    vce = _load_duckdb_table()  # noqa: F841
-    errors = []
-    for report_year, length in duckdb.query(
-        "SELECT report_year, COUNT(*) FROM vce GROUP BY ALL"
-    ).fetchall():
-        if (expected_length := row_counts_by_year[report_year]) != length:
-            errors.append(
-                f"Expected {expected_length} for report year {report_year}, found {length}"
-            )
-    if errors:
-        logger.warning(errors)
-        return AssetCheckResult(
-            passed=False,
-            description="One or more report years have unexpected length",
-            metadata={"errors": errors},
-        )
-    return AssetCheckResult(passed=True)
-
-
-@asset_check(
-    asset=out_vcerare__hourly_available_capacity_factor,
-    blocking=True,
-    description="Check for unexpected nulls.",
-)
-def check_nulls() -> AssetCheckResult:
```

Comment:
Translated all the rest of these checks into `dbt_expectations` checks if they hadn't been migrated already.